modules/offers/server/controllers/offers.server.controller.js

```diff
@@ -485,21 +485,20 @@ exports.list = function (req, res) {
   }
 
   // Some of the filters are based on `user` schema
-  if (filters.hasArrayFilter('languages') || filters.hasArrayFilter('tribes') || filters.hasObjectFilter('seen')) {
-    query.push({
-      $lookup: {
-        from: 'users',
-        localField: 'user',
-        foreignField: '_id',
-        as: 'user'
-      }
-    });
-    // Because above `$lookup` returns an array with one user
-    // `[{userObject}]`, we have to unwind it back to `{userObject}`
-    query.push({
-      $unwind: '$user'
-    });
-  }
+  query.push({
```

Comment:
Noting that these days we're always sending `seen` in queries, so we're anyway doing a lookup to `users` collection.